packages/components/src/search-control/style.scss

```diff
@@ -41,8 +41,15 @@
 		height: $grid-unit-50;
 	}
 
-	&.is-size-compact input[type="search"].components-search-control__input {
-		height: $grid-unit-40;
+	&.is-size-compact {
+		input[type="search"].components-search-control__input {
+			height: $grid-unit-40;
+			padding-left: $grid-unit-10;
```

Comment:
Should we also update the input field's `padding-right` to a smaller size? (ie. `$grid-unit-50`)?